website/source/docs/platform/k8s/dns.html.md

```diff
@@ -40,6 +40,38 @@ of the `consul-dns` service changes, then it must be updated to continue
 working. This can happen if the service is deleted and recreated, such as
 in full cluster rebuilds.
 
+## CoreDNS Configuration
+
+If you are using CoreDNS instead of kube-dns in your Kubernetes cluster, you will
+need to update your existing `coredns` ConfigMap in the `kube-system` namespace to
+include a proxy definition for `consul` that points to the cluster IP of the 
+`consul-dns` service.
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+  name: coredns
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        <Existing CoreDNS definition>
+    }
+    consul {
+      errors
+      cache 30
+      proxy . <consul-dns service cluster ip>
+    }
+```
+
+-> **Note:** The consul proxy can only point to a static IP. If the cluster IP
+of the `consul-dns` service changes, then it must be updated to continue
```

Comment:
Updated where? Updated so that it is static? Or updated everything time the IP changes?

Reply:
Updated to be a bit more explicit about what needs to change in this situation.